repo/content/sessions.go

```diff
@@ -0,0 +1,179 @@
+package content
+
+import (
+	"context"
+	cryptorand "crypto/rand"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/kopia/kopia/internal/gather"
+	"github.com/kopia/kopia/repo/blob"
+)
+
+// BlobIDPrefixSession is the prefix for blob IDs indicating active sessions.
+// Each blob ID will consist of {sessionID}.{suffix}.
+const BlobIDPrefixSession blob.ID = "s"
+
+// SessionID represents identifier of a session.
+type SessionID string
+
+// SessionInfo describes a particular session and is persisted in Session blob.
+type SessionInfo struct {
+	ID             SessionID `json:"id"`
+	StartTime      time.Time `json:"startTime"`
+	CheckpointTime time.Time `json:"checkpointTime"`
+	User           string    `json:"username"`
+	Host           string    `json:"hostname"`
+}
+
+var (
+	sessionIDEpochStartTime   = time.Date(2000, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Is the year intended to be `2020` ?

Reply:
it is arbitrary, really, but we have some hardcoded timestamps before 2020 in code already.

Reply:
I mean, the comment below says `2020-01-01`, but `sessionIDEpochStartTime` is set to `2000-01-01`